TwilioVerifySDK/TwilioSecurity/Sources/Storage/AuthenticatedSecureStorage.swift

```diff
@@ -28,7 +28,7 @@ public typealias ErrorBlock = (Error) -> ()
 ///:nodoc:
 public protocol AuthenticatedSecureStorageProvider {
   func save(_ data: Data, withKey key: String, authenticator: Authenticator, success: @escaping EmptySuccessBlock, failure: @escaping ErrorBlock, withServiceName service: String?)
-  func get(_ key: String, authenticator: Authenticator, success: @escaping SuccessBlock, failure: @escaping ErrorBlock)
+  func get(_ key: String, authenticator: Authenticator, attempts: Int?, success: @escaping SuccessBlock, failure: @escaping ErrorBlock)
```

Comment:
Do we need a retry for this storage?

Reply:
Actually the **AuthenticatedSecureStorage** is the one that require this change right now, since it makes use of the **get** while doing biometric verifications. The one in the **SecureStorage** could be optional, but it could be useful in some scenarios.